test/Project.toml

```diff
@@ -6,3 +6,6 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
 Aqua = "0.7"
+Documenter = "0.27, 1.0"
```

Comment:
Why `1.0`?

Reply:
for doctests, nothing breaking should've changed. But yeah, consistency is nice...